registry_test.go

```diff
@@ -144,3 +130,28 @@ func BenchmarkRegistry(b *testing.B) {
 		_, _ = r.LookupSubnets(ctx)
 	}
 }
+
+func getTestRegistry(cacheMisses *uint32) *Registry {
```

Comment:
```suggestion
func newTestRegistry(cacheMisses *uint32) *Registry {
```

Reply:
Constructors are usually named new-something in Go. "get" feels like we're taking it from somewhere, which is not the case here.

Reply:
Fixed.